test/routable_component_routes.js

```diff
@@ -63,31 +63,31 @@ describe('RoutableComponentRoutes', function() {
   });
   /** @test {RoutableComponentRoutes#constructor} */
   context('constructed with', function() {
-    subject(function() { return new RoutableComponentRoutes(this.routing_classes) });
+    subject(function() { return new RoutableComponentRoutes(this.routingClasses) });
     context('single routing classes', function() {
-      lazy('routing_classes', function() { return MyRouting });
+      lazy('routingClasses', function() { return MyRouting });
```

Comment:
__[eslint]__
* Error - 'lazy' is not defined. (no-undef)
* Warning - Too many nested callbacks (4). Maximum allowed is 3. (max-nested-callbacks)
* Warning - Unexpected function expression. (prefer-arrow-callback)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->